src/app/components/core/configuration-manager/view-config-item/view-config-item.component.ts

```diff
@@ -459,6 +461,9 @@ export class ViewConfigItemComponent implements OnInit, OnChanges, OnDestroy {
             .replace(new RegExp(key, 'g'), `'${configValue}'`);
         }
       }
+      if (configItem.hasOwnProperty('mandatory') && configValue.trim().length === 0) {
```

Comment:
we should also check if mandatory is true, otherwise this will trigger even if it is set to false.

Reply:
and this will also apply to Advanced Config as well?

Reply:
done